lib/dark_worlds_server/runner_supervisor/runner.ex

```diff
@@ -266,7 +248,7 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
 
   @impl true
   def terminate(_reason, state) do
-    player_count = length(state.game_state.players) - state.bot_count
+    player_count = Enum.count(state.game_state.players) - state.bot_count
```

Comment:
nit: Why this change?

Reply:
nvm, I just saw the previous comment.

Reply:
You are probably right, made the change as part of bb8dee8